plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/model/MySQLDialect.java

```diff
@@ -43,157 +42,183 @@
 public class MySQLDialect extends JDBCSQLDialect implements SQLDialectSchemaController {
 
     public static final String[] MYSQL_NON_TRANSACTIONAL_KEYWORDS = ArrayUtils.concatArrays(
-        BasicSQLDialect.NON_TRANSACTIONAL_KEYWORDS,
-        new String[]{
-            "USE", "SHOW",
-            "CREATE", "ALTER", "DROP",
-            SQLConstants.KEYWORD_EXPLAIN, "DESCRIBE", "DESC"}
-    );
-
-    private static final String[] ADVANCED_KEYWORDS = {
-        "AUTO_INCREMENT",
-        "DATABASES",
-        "COLUMNS",
-        "ALGORITHM",
-        "REPAIR",
-        "ENGINE"
-    };
-
-    public static final String[][] MYSQL_QUOTE_STRINGS = {
-            {"`", "`"},
-            {"\"", "\""},
-    };
-
+            BasicSQLDialect.NON_TRANSACTIONAL_KEYWORDS, new String[] { "USE", "SHOW", "CREATE", "ALTER", "DROP" });
```

Comment:
Why did you delete DESCRIBE and DESC keywords?

Reply:
because BasicSQLDialect.NON_TRANSACTIONAL_KEYWORDS already has DESCRIBE, DESC and EXPLAIN I removed it.

Reply:
Ok, thanks